### PR TITLE
Plugin stabilization - as discussed at the last WET roadmap meeting

### PR DESCRIPTION
**Pull request by duboisp**

### CODE REVIEW

File: site/pages/docs/ref/bgimg/bgimg-en.hbs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+	"categoryfile": "plugins",
+	"description": "Apply a background image the URL given in paramenter or as per the srcset information.",
+	"altLangPrefix": "bgimg",
+	"dateModified": "2020-12-09"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<h2>Purpose</h2>
+<p>Apply a background image the URL given in paramenter or as per the srcset information.</p>
```
Comment (RobJohnston) [inline site/pages/docs/ref/bgimg/bgimg-en.hbs:15]: "a background image _from_ the given URL"?
paramenter  => parameter

File: site/pages/docs/ref/bgimg/bgimg-en.hbs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+<h2>Use when</h2>
+<ul>
+	<li>Display a background decorative image</li>
```
Comment (RobJohnston) [inline site/pages/docs/ref/bgimg/bgimg-en.hbs:19]: Display a background decorative image => Display a decorative background image

File: site/pages/docs/ref/bgimg/bgimg-en.hbs [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+</ul>
+
+<h2>Guidance for publisher</h2>
+<ul>
+	<li>Ensure the text contrast remain valid in the case where the image don't load or take time to load</li>
```
Comment (RobJohnston) [inline site/pages/docs/ref/bgimg/bgimg-en.hbs:24]: Ensure the text contrast remains valid in the case where the image doesn't load or takes time to load

File: site/pages/docs/ref/wb-randomize/wb-randomize-en.hbs [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	"language": "en",
+	"category": "Plugins",
+	"categoryfile": "plugins",
+	"description": "Randomly picks one of the child element.",
+	"altLangPrefix": "wb-randomize",
+	"dateModified": "2020-12-09"
+}
+---
+<h2>Purpose</h2>
+<p>Randomly picks one of the child element.</p>
```
Comment (RobJohnston) [inline site/pages/docs/ref/wb-randomize/wb-randomize-en.hbs:13]: element => elements.

File: src/plugins/bgimg/bgimg-en.hbs [hunk 5]
```diff
@@ -0,0 +1,66 @@
+---
+{
+	"title": "Background image",
+	"language": "en",
+	"category": "Plugins",
+	"description": "Apply a background image the URL given in paramenter or as per the srcset information.",
```
Comment (RobJohnston) [inline src/plugins/bgimg/bgimg-en.hbs:6]: "a background image _from_ the given URL"?
paramenter  => parameter

File: src/plugins/bgimg/bgimg-en.hbs [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+	"tag": "bgimg",
+	"parentdir": "bgimg",
+	"altLangPrefix": "bgimg",
+	"dateModified": "2020-12-08"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<p>Apply a background image the URL given in paramenter or as per the image group (<code>srcset</code>) information.</p>
```
Comment (RobJohnston) [inline src/plugins/bgimg/bgimg-en.hbs:15]: "from the given URL"?
paramenter  => parameter

File: src/plugins/bgimg/bgimg-en.hbs [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+<p>Note: Examples bellow use the new CSS recently introduced in wet-boew: <code>bg-center</code>, <code>bg-cover</code> and <code>bg-norepeat</code></p>
```
Comment (RobJohnston) [inline src/plugins/bgimg/bgimg-en.hbs:17]: bellow => below

File: src/plugins/bgimg/bgimg-en.hbs [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+	<summary>Source code</summary>
+	<pre><code>&lt;div class="bg-cover bg-center well" data-bgimg="../../demos/tabs/img/investinourfuture.jpg"&gt;
+		&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+			&lt;h4 class="mrgn-tp-md"&gt;Heading&lt;/h4&gt;
+			&lt;p&gt;Paragraph&lt;/p&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;</code></pre>
+</details>
+
+<p>Note: Eventually, the javascript associated with this feature would be replacable by a CSS4 selector.</p>
```
Comment (RobJohnston) [inline src/plugins/bgimg/bgimg-en.hbs:38]: replacable  => replaceable

File: src/plugins/wb-randomize/wb-randomize-en.hbs [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+	"description": "Randomly picks one of the child element.",
+	"tag": "randomize",
+	"parentdir": "wb-randomize",
+	"altLangPrefix": "wb-randomize",
+	"dateModified": "2020-12-09"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<p>Randomly picks one of the child element.</p>
```
Comment (RobJohnston) [inline src/plugins/wb-randomize/wb-randomize-en.hbs:15]: element => elements

**Review by RobJohnston — COMMENTED** [review]
Small changes to English spelling and grammar.

**Review by GormFrank — CHANGES_REQUESTED** [review]
Add plugins to the `<ul>` in the **site/pages/docs/ref/plugins-en.hbs** file and same in French.

**Review by GormFrank — APPROVED** [review]

**Pull request merged**